### ISSUE TITLE
How to divide train, valid and test set?

### ISSUE THREAD

**Issue by Jacklisss** [zjukg/FedE#3]
”The proportion for training/validation/testing triples is 0.8/0.1/0.1.“I found that these three sets have repeated relationships and repeated entities. I understand that training and verification sets should belong to the same category, while test sets should belong to another category.I did an interesting experiment, and the results showed that it worked well in the valid set, but in the end I got a particularly bad result in the test set. If the three of them were in the same part, why did this happen?

**Comment by AnselCmy** [comment]
As described in our paper, In order to evaluate our
FedE in the federated setting, we create several new datasets in the
federated setting by distributing triples into clients. Specifically, according to Definition 2.2, we randomly select relations into clients
and distribute triples into clients according to selected relations.
We randomly split FB15k-237 into 3,5,10 clients as FB15k-237-Fed3,
-Fed5, -Fed10 and NELL-995 into 3 clients as NELL-995-Fed3

**Comment by AnselCmy** [comment]
As described in our paper, " In order to evaluate our FedE in the federated setting, we create several new datasets in the
federated setting by distributing triples into clients. Specifically, we randomly select relations into clients and distribute triples into clients according to selected relations. We randomly split FB15k-237 into 3,5,10 clients as FB15k-237-Fed3,
-Fed5, -Fed10 and NELL-995 into 3 clients as NELL-995-Fed3". Furthermore, please give me the details of your experiments, thanks!

**Comment by Jacklisss** [comment]
Thank you for your reply. I also want to ask whether the training set, verification set and test set are divided by triples randomly selected

**Comment by AnselCmy** [comment]
Yes, we divided them randomly as mentioned in our paper.
